addons/mail/static/src/core/common/composer.xml

```diff
@@ -115,6 +115,15 @@
     </button>
 </t>
 
+    <t t-name="mail.Composer.suggestionSpecial">
+        <strong class="px-2 py-1 align-self-center flex-shrink-0 text-truncate">
+            <t t-esc="option.name"/>
+        </strong>
+        <em class="text-600 text-truncate align-self-center">
+            (<t t-esc="option.description"/>)
+        </em>
```

Comment:
Remove brackets `()`. On partner this is used to de-emphasis email address from the name of user, but here with `@everyone` the description is an important piece of suggestion, like commands